### PR TITLE
Merge differences resolved, working back up to state of art

### PR DESCRIPTION
**Pull request by benfrancis314**
Timing was just due to minor rearrangement of timer vs. movedown; all updates should be operable in this version

### CODE REVIEW

File: extension/content_script.js [hunk 1]
```diff
@@ -91,28 +91,48 @@ function move(type) { // Note: I have combined the "moveUp" and "moveDown" funct
 	} else {  // When the button is held down
 		if (!timer) {
 			(function repeat() { // Allows speed to be updated WHILE moving
-				timer = setTimeout(repeat, speed_adj);
+				console.log(speed_adj);
```
Comment (sjonany) [inline extension/content_script.js]: Still need this for debugging?

File: extension/content_script.js [hunk 2]
```diff
@@ -140,55 +160,33 @@ function setTracker(type) {
...
-			$('html, body').animate(
-				// Leave some vertical margin before the container.
-				{scrollTop: (markedTopAbsoluteOffset - 200)},
-				500, /* duration(ms) */
-				function() {
-					isScrolling = false;
-				}
-			);
-		}
+		scroll()
```
Comment (sjonany) [inline extension/content_script.js]: Semicolon

**Review by sjonany — CHANGES_REQUESTED** [review]
Looks good overall! Just minor nits.

File: extension/content_script.js (hunk 1 above)
Comment (benfrancis314) [reply extension/content_script.js]: Sounds good! Will fix then re-pull request soon

**Comment by sjonany** [comment]
LGTM

**Pull request merged**